### ISSUE TITLE
How to convert Response Body Attribute value to Numeric variable

### ISSUE THREAD

**Issue by krithigagupta** [getinsomnia/insomnia#2264]
**What is your question?**
A clear and concise description of what you are wanting to ask. Ex. I'm always frustrated when [...]
I want to understand how to convert a response body attribute to numeric. I have two requests that I have chained together. First request returns a response body attribute, which is used in the second request. But when I submit the second request, I get the below error
"TypeError: Int cannot represent non-integer value: \"11832\"",
            "    at GraphQLScalarType.coerceInt [as parseValue] 

The value from the first request is numeric, however it is getting passed as a string.
![image](https://user-images.githubusercontent.com/32104725/84074087-9a2cbb00-a997-11ea-8e66-1575ab2caa06.png)

**Comment by k-aziz** [comment]
It looks like you're surrounding the Response Attribute tag with double quotes. Have you tried removing them?

**Comment by krithigagupta** [comment]
Yes. I did. It didn't work with that either. Are there any built-in functions that I could use to convert to numeric?

**Comment by krithigagupta** [comment]
Any updates on this one, or any workaround to pull non-string values (number, array, etc.) from previous responses?

**Comment by solarmosaic-kflorence** [comment]
@krithigagupta looks the same as https://github.com/Kong/insomnia/issues/973

**Comment by rkenshin** [comment]
Is there an update to this issue. I'm having similar issue stilt with version 2021.7.2
